### PR TITLE
Issue #654: Refactor unit tests for extension.ts file

### PR DESCRIPTION
**Pull request by katelynienaber**
Refactoring of extension.ts unit tests file.

NOTE: File `extension-beforeRefactor.unit.test.ts` contains tests that should be moved to other files
- Some have been moved already in other PRs.
- Some will be moved when their test files are refactored in the (near) future.

Related to issue [#654](https://github.com/zowe/vscode-extension-for-zowe/issues/654)

### CODE REVIEW

File: __tests__/__unit__/extension-beforeRefactor.unit.test.ts [hunk 1]
```diff
@@ -0,0 +1,4754 @@
...
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
```
Comment (jellypuno) [inline __tests__/__unit__/extension-beforeRefactor.unit.test.ts:11]: beforeRefactor? Why do you need this?

Comment (katelynienaber) [reply __tests__/__unit__/extension-beforeRefactor.unit.test.ts:11]: Explained in PR description.

**Comment by katelynienaber** [comment]
> I would assume that all tests that were removed in extension.unit.test.ts was transferred to a different location.

They still exist in beforeRefactor. Please read PR description.

**Comment by jellypuno** [comment]
> > I would assume that all tests that were removed in extension.unit.test.ts was transferred to a different location.
> 
> They still exist in beforeRefactor. Please read PR description.

I still don't know why do you have beforeRefactor. If I am an outsider, I shouldn't know or care that there is a refactoring happened. There should just be one file. my POV

**Comment by katelynienaber** [comment]
> > > I would assume that all tests that were removed in extension.unit.test.ts was transferred to a different location.
> > 
> > 
> > They still exist in beforeRefactor. Please read PR description.
> 
> I still don't know why do you have beforeRefactor. If I am an outsider, I shouldn't know or care that there is a refactoring happened. There should just be one file. my POV

The functions need to be moved to other files which haven't been refactored yet. If I delete the file, the tests will be deleted completely.

So probably the best option is to wait on refactoring this file until all other files are refactored, and the tests are moved out of extension.unit.test.ts. Then extension.unit.test.ts can be refactored and the old file deleted completely...

If you agree then I will delete this PR and save it for later.

**Pull request closed without merging**